### DISCUSSION TITLE
Dev mode - aileron issue

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4856] [General]
Hi, 

does anybody else have issues with the ailerons on ground? Moving the stick to the right and hold them in the right position (for example), the ailerons won´t stay at that position. You can also see this on the cross in the PFD..

https://youtu.be/k9VWsDg4Atc

greets Klaus

**Comment by ExampleWasTaken** [comment]
There is an issue with the SDK and ailerons at the moment. I am pretty sure that your issue is of the same kind.

**Reply by CptBayer** [reply]
ah ok.... cause I never saw this before..... 

Cheers EWT !
